### ISSUE TITLE
Bug Fix: Hex Editor no open when selected at the PopUp Menu on the Query Window.

### ISSUE THREAD

**Issue by JeymyAV** [avan06/PS4CheaterNeo#30]
I don't know if I am doing something wrong here but based in the below picture:
![image](https://github.com/avan06/PS4CheaterNeo/assets/9624829/abf7c120-6e2c-45d2-a757-187da9a834b3)

When I click in the Hex Editor PopUp menu then it should be open with the selected section range?

Because what happen to me is that the popup menu close and the Hex Editor is not open...

Is this behavior correct or is  it a bug or what I am doing wrong? 

Version tested is Neo 0.9.9.3 but I think that it happened on the all previous one too (from 0.9.8.1 until this last one).

**Comment by JeymyAV** [comment]
Oh, I find out what I was did wrong, I have to select the column number section instead of the row with the address...

Well, it is my bad... but should be possible to work the Check PopUp option and the Hex Editor pop up option selecting the row and no only the column with the number section?

**Comment by avan06** [comment]
Because I didn't want checkboxes to be automatically selected when clicking on any column, I restricted the control to the first column position. However, in version 0.9.9.1, a new option called "`SectionViewFullRowSelect`" was added. Enabling it allows you to click on any column, which seems to be what you're looking for.

**Comment by JeymyAV** [comment]
Oh, never see this option "SectionViewFullRowSelect" before... Ok enabled and testing...

I see the difference, now it works, I can select the row and the HexEditor is open when I choose the option in the popup menu, but as you said, this has one side-effect, the section is automatically selected.. This behavior is a bit weird...

Humm, Why is automatically checked the section when you click in the row? cannot it be avoid it?

Normally most of Windows application you make the selection of the row when you do double click, no with a single click like is happen now.

On my opinion with this option enable, I would prefer that when the user click on the row, the section **_only should be mark_** with the popup menu option "Check" or using _**double click**_ and not single click of the button mouse like now, but it is up to you decide how it should be work...

If you say that this is the design behavior of the tool and you are no going to change it, for me is ok and I will close the issue...

**Comment by Pharaoh2k** [comment]
Yeah, I agree with @JeymyAV 
If the checkbox can be set to only be checked on double-click, rather than single click which is meant only for highlighting, that's a lot better.

**Comment by avan06** [comment]
I later noticed the reason for the checkboxes getting selected. It was due to the switch to VirtualMode to improve the SectionView processing speed. VirtualMode has limited support for checkboxes, and some behaviors changed as a result. Now, in version [0.9.9.6-beta](https://github.com/avan06/PS4CheaterNeo/releases/tag/0.9.9.6-beta), I have improved this. Even with "`SectionViewFullRowSelect`" enabled, clicking on any column will not cause checkboxes to be selected. Only clicking on the first column will select the checkbox. Thank you.
